doc-Community_Collaboration_Guide/community/master.adoc

```diff
@@ -1,14 +1,11 @@
 = Community Contribution Guide
+Red Hat Customer Content Services
 :imagesdir: images
-:vernum: 1.0
 :doctype: book
 :toc: left
 :toclevels: 3
-:numbered!:
 :experimental:
 
-include::common/attributes/community.adoc
```

Comment:
Why have you decided to remove the attributes include in favor of putting them in the file?  Using the attributes file is pretty common in our AsciiDoc guides.

Reply:
Note that `common/attributes/community.adoc` does not exist, so I just removed a broken include. If you think we should have a separate file for attributes (although I believe this is only useful when you have multiple guides), feel free to do add it in a separate PR.

Reply:
Well, I should have totally checked on that!! I agree with what you've done. I think we should probably create an attributes file in a common folder and include that for both guides.  I'm not too fussed about it though if you think that's not an efficient way to handle things.  For now I approve of this change.